src/Fist/Http/tests/RequestTest.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+use Fist\Http\Request;
+use Fist\Testing\TestCase;
+
+class RequestTest extends TestCase
```

Comment:
__[phpmd]__
* The class RequestTest has 16 public methods. Consider refactoring RequestTest to keep number of public methods under 10.

<!-- Comment by [SideCI](https://sideci.com) -->